Docker-compose/README.md

```diff
@@ -12,6 +12,8 @@
 
 ## Requirements
 
+The following instructions and provided scripts are intended to be used on a RedHat or Ubuntu based system. Success on Windows based system is currenlty not guaranteed.
```

Comment:
```suggestion
The following instructions and provided scripts are intended to be used on a GNU/Linux based system. Success on Windows based system is currently not guaranteed.
```

Reply:
applied in 0ac633a5385c80a5f77ae3e4a01a6bcc087b831d